astrometry/include/PixelMap.h

```diff
@@ -17,8 +17,8 @@
 #ifdef USE_YAML
 #include "yaml-cpp/yaml.h"
 #endif
-#include "astshim.h"
-
+#include "astshim/Object.h"
+#include "astshim/Mapping.h"
```

Comment:
Is this just to reduce what is being included?

Reply:
Yes, to only include what is used.